TheMovieDB/Entities/Movie.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Movie.swift
+//  TheMovieDB
+//
+//  Created by Angela Lee on 27/10/2022.
+//
+
+import Foundation
+
+class Movie {
+    var title: String?
+    var path: String?
```

Comment:
Maybe a better descriptive name for this variable would be "posterPath" or "imagePath"

Reply:
Solved 02c0bb086ee274bead0a291207ba7d1ef5568abd